### PR TITLE
fix(mobile): keep empty album as selected target

### LINKED ISSUE #7711: Mobile App does not correctly handle the selected backup folder, if it is/becomes empty.
### The bug

If I empty the selected backup folder on my mobile phone, then Immich App will kick this folder out of the backup folder list.
Like the screenshots show in the below.

Now "Camera" is selected and there are 4 photos in it.
Screenshot_20240307-183341~2
Screenshot_20240307-183351~2

Then I have deleted all the photos in "Camera"
Screenshot_20240307-183437~2

even you can't select a folder as backup folder if it is currently empty.
Screenshot_20240307-183457_Mosaic_19_59_49

### The OS that Immich Server is running on

Docker Desktop on Windows

### Version of Immich Server

v1.97.0

### Version of Immich Mobile App

v1.97.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
nothing special, default version with customized port number
```

### Your .env content

```Shell
nothing special, default version with customized path
```

### Reproduction steps

```bash
1.just delete all the photos in the selected backup folder
```

### Additional information

This bug is not just introduced recently, it exists already long time. But I can't remember the start-point.

### PR DESCRIPTION
**Pull request by alextran1502**
Fixes #7711

### CODE REVIEW

File: mobile/lib/repositories/album_media.repository.dart [hunk 1]
```diff
@@ -14,7 +14,6 @@ class AlbumMediaRepository implements IAlbumMediaRepository {
     final List<AssetPathEntity> assetPathEntities =
         await PhotoManager.getAssetPathList(
       hasAll: true,
-      filterOption: FilterOptionGroup(containsPathModified: true),
```
Comment (mertalev) [inline mobile/lib/repositories/album_media.repository.dart:17]: What does this do?

Comment (alextran1502) [reply mobile/lib/repositories/album_media.repository.dart:17]: Basically it excludes all of the empty album

**Review by mertalev — APPROVED** [review]

**Comment by IvanCharcos** [comment]
This is still an issue as of today. I am using latest Immich server 1.125.2.build.178 and Immich App 1.125.2.
Albums still disappear from "Backup Albums list" once I delete all photos from device after being backed-up.

**Comment by nchevsky** [comment]
> This is still an issue as of today. I am using latest Immich server 1.125.2.build.178 and Immich App 1.125.2. Albums still disappear from "Backup Albums list" once I delete all photos from device after being backed-up.

The fix was later reverted in #15461, and issue #7711 has been reopened.

**Pull request merged**